reana_client/api/client.py

```diff
@@ -669,7 +750,15 @@ def get_workflow_specification(workflow, access_token):
 def delete_workflow(workflow, all_runs: bool, workspace: bool, access_token: str):
     """Delete a workflow.
 
-    Please note that the workspace will be always deleted, even if ``workspace`` is set to ``False``.
+    :param workflow: name or id of the workflow.
+    :param all_runs: whether to delete all the runs of the workflow.
+    :param workspace: whether to delete the workspace of the workflow.
+    :param access_token: access token of the current user.
+
+    :return: a dictionary that cointains info about the deleted workflow (``workflow_id``, ``workflow_name``,
+             ``status``, ``user``), and a ``message`` key.
+
+    Please note that the workspace will always be deleted, even if ``workspace`` is set to ``False``.
```

Comment:
What do you think about putting this (_Please note ..._)  just after the first description line, and before the parameters' list?

Reply:
At first I thought that it was better to put this note after explaining the parameters, but after a quick look online, it seems common practice to put the whole textual description, even referring to parameters, before the actual list, so I changed it!